base/compiler/ssair/inlining.jl

```diff
@@ -873,8 +891,22 @@ function validate_sparams(sparams::SimpleVector)
     return true
 end
 
+function has_fcalls(m::Method)
```

Comment:
Shouldn't this check be conservative and return `true` in unanalyzable case (and maybe should be renamed to `may_has_fcalls`)?